test/e2e/utils/record-helpers.js

```diff
@@ -237,6 +237,24 @@ exports.testPresentation = function (tableParams) {
             expect('Encountered an error').toBe('Please check the log', 'Inside catch block');
         })
     });
+	
+	it("visible column related table with inline inbound fk should display 'None' in markdown disply mode if no data was found.",function(){
+        chaisePage.waitForElement(element(by.id('entity-booking'))).then(function(){
+            return browser.executeScript("return $('#entity-booking .btn-group .delete-action-button')");
+        }).then( function(deleteButtons){
+			// browser.pause();
+			for(var i = 0; i < deleteButtons.length; i++){
+				deleteButtons[0].click();
+				browser.executeScript("return $('#delete-confirmation').click()");
+			}
+            return browser.executeScript("return $('a.toggle-display-link').click()");
+        }).then(function(){
+            expect(element(by.id('entity-4-markdown')).getText()).toBe('None',"Incorrect text for empty markdown!");
```

Comment:
Add a wait condition before this line to make sure the markdown display is being shown first.